src/book.py

```diff
@@ -86,7 +86,23 @@ def _read_binance(self, file_path: Path) -> None:
             # Skip header.
             next(reader)
 
-            for _utc_time, account, operation, coin, _change, remark in reader:
+            for rowlist in reader:
+                if version == 1:
+                    _utc_time, account, operation, coin, _change, remark = reader
+                if version == 2:
```

Comment:
This should read `elif version == 2:`. Otherwise, if the file is version 1, it would trigger the warning/error from the else block below.